### PR TITLE
Add Ticket

### PR DESCRIPTION
**Pull request by AsalaKM**
**Add Ticket page**
 Allow the logged employee to add new ticket In both of type (support or loan)
**#7**

### CODE REVIEW

**Review by OnsNader — APPROVED** [review]

**Comment by FarahZaqout** [comment]
This is still work in progress. PR name should reflect that.
@OnsNader Don't just approve for the sake of approval. The file has conflicts with master.

**Review by SalwaMugh — APPROVED** [review]

File: public/js/addTicket.js [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+      .catch(() => {
+        error.textContent = 'THERE IS ERROR';
+      });
+  }
+});
+
+// ADD SUPPORT TICKET
+
+const empName = 'MohAmmed';
+const empDep = 'information technology';
```
Comment (FarahZaqout) [inline public/js/addTicket.js]: This was done for testing purposes. It should be replaced with the actual field value.

File: public/js/addTicket.js [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+const problemdesc = document.querySelector('#pdesc-support');
+const btnAddSupport = document.querySelector('#btn-add-support');
+const Msg = document.querySelector('.msg-support');
+const errMsg = document.querySelector('.error-support');
+
+btnAddSupport.addEventListener('click', (e) => {
+  e.preventDefault();
+  if (empName && empDep && pType && tSub && ticketType && ticketPriority
+    && ticketDuration && problemdesc) {
+    const empC = empName;
```
Comment (FarahZaqout) [inline public/js/addTicket.js]: empC, depC...etc is not descriptive.

File: public/js/addTicket.js [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+        error.textContent = 'THERE IS ERROR';
+      });
+  }
+});
+
+// ADD SUPPORT TICKET
+
+const empName = 'MohAmmed';
+const empDep = 'information technology';
+const pType = document.querySelector('#ptype-support');
```
Comment (FarahZaqout) [inline public/js/addTicket.js:85]: use descriptive names please.

**Review by FarahZaqout — CHANGES_REQUESTED** [review]

**Review by FarahZaqout — APPROVED** [review]

**Pull request merged**